modules/spo-applying-profiles.adoc

```diff
@@ -36,6 +36,7 @@ kind: Pod
 metadata:
   name: test-pod
 spec:
+  # TODO: How to update this example? Shoudl it be added on the container below too?
```

Comment:
@s-urbaniak Another one that has existing securityContext set, what should we do here?

Reply:
add the other changes but keeping the seccompProfile original

Reply:
I added the securityContext to the container. And then at the top I just added `runAsNonRoot: true`. Was that what you meant?